### ISSUE TITLE
TimeInput filters don't work after build

### ISSUE THREAD

**Issue by Derranion** [epicmaxco/vuestic-ui#1544]
**Vuestic-ui version:** 1.3.4

**Steps to reproduce:**
[Code sandbox example.](https://codesandbox.io/s/dazzling-silence-g1hwtr?file=/src/App.vue:34-113)
1. Create a new project
2. Add `TimeInput` there
3. Add minutes or hours filter to this `TimeInput`

Code example:
```
  <va-time-input v-model="value" ampm :minutesFilter="(m) => m % 10 === 0" />
  <va-time-picker v-model="value" ampm :minutesFilter="(m) => m % 10 === 0" />
```
**What is the expected behavior?**
`Filters` should apply to the `TimeInput` same way as it is applied to the `TimePicker`.
In our case it should display only 10, 20, 30 ... minutes
**What is the current behavior?**
TimeInput filters work locally but do not work after the build using Vite or on codesandbox. (didn't test others)
**Other information**
[Issue raised in our discord initially](https://discord.com/channels/457593300799717377/771405157527191585/950023220650725437).

**Comment by Derranion** [comment]
Update:
Any props that are passed from `TimeInput` to `TimePicker` (used as a dropdown content inside of the `TimeInput`) are not working after build.

**Comment by Derranion** [comment]
It was caused by this line of code in the `1.3.4` version:
`const timePickerProps = computed(() => filterComponentProps(props, extractComponentProps(VaTimePicker)))
`

The reason behind it was in wrapping already existed computed by the second one:
![vueUnpacksOnlyOneComputedByValue](https://user-images.githubusercontent.com/7745899/157457773-4392cc54-93fd-469d-b300-610ad7b64fc1.jpg)

Vue wasn't able to parse this properly and rolled back to default values.

I've checked it using new Vite project and new build of recently updated Vuestic (`dev` branch) and it was fixed by random refactor already.

That's why it will be fixed after the new release, `on-hold` till `1.4.0` just to check it in the final version.
